performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -17,20 +17,76 @@
 
 #include <sys/time.h>
 #include <sys/resource.h>
-
+#include <boost/uuid/uuid_io.hpp>
 #include <chrono>
 #include <sstream>
 #include <string>
 
 #include "../utilities/statistics_tracker.hpp"
+#ifdef ODB_FOR_SQL_ENABLED
+  #include <odb/core.hxx>
+  #include "experiment_configuration.hpp"
+#endif
 
 namespace performance_test
 {
 
 /// Outstream operator for timeval to seconds (double).
 std::ostream & operator<<(std::ostream & stream, const timeval & e);
+#ifdef ODB_FOR_SQL_ENABLED
+class RusageTracker
+{
+public:
+  void setValues(rusage sys_usage) const
```

Comment:
This should not be const, in fact, this should be the constructor and then you set the members in the constructor initialization list.

Reply:
ok, thanks, done